play.py

```diff
@@ -117,7 +118,8 @@ def note2freq(x):
 
 
 if __name__ == '__main__':
-
+    
+    pyautogui.PAUSE = 0
```

Comment:
Actually might even be better with 0.05 (avoids game missing keypresses if you're in a crowded environment like limes for example and don't have a perfect framerate)